internal/controllers/clustertemplate_controller.go

```diff
@@ -98,6 +102,7 @@ func (r *ClusterTemplateReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 	_ = log.FromContext(ctx)
 	result = doNotRequeue()
 
+	time.Sleep(100 * time.Millisecond)
```

Comment:
Could we add a brief description here for this pause?

Reply:
Fixed